### PR TITLE
Reformat the HTML in a bunch of my blog posts

### PR DESCRIPTION
**Pull request by realsimonwu**
When I tried to view blogposts I transcribed using the development server I noticed that they weren't actually being rendered as HTML. I reformatted a bunch of the blogposts I transcribed to make it actually render.

### CODE REVIEW

File: _digest/january-2021/will-ai-drive-healthcare-workers-into-extinction.mdx [hunk 1]
```diff
@@ -6,229 +6,139 @@ summary: Artificial intelligence is already disrupting global economics systems
...
+					Radiologists are trained to read medical imaging technologies such as MRIs, CT Scans, and X-rays to detect the presence of disease symptoms.
+					Likewise, dermatologists determine whether skin abnormalities are benign or cancerous.
+					Such work is highly pattern intensive which is one of AI’s greatest strengths.
+					So much so that in some cases AI is currently better at diagnosing breast cancer than radiologists by the sheer ability to reference a greater database of knowledge.
+					However, radiologists still prescribe appropriate treatment plans and counsel patients; a task AI simply cannot do effectively.</p>
+				<p>Such can also be said about epidemiology and pathology, where AI predictive models are already being used to predict and monitor disease outbreaks and disease progression respectively.</p>
+				<p>Furthermore, though some menial tasks done by pharmacists can be automated, humans will still be required to ensure patients are compliant and consistent with their medications.</p>
+				<!-- (If you want to learn how to be a Pharma D, click here to read an article by Dhruv Dhall). -->
```
Comment (hirundinidae) [inline _digest/january-2021/will-ai-drive-healthcare-workers-into-extinction.mdx]: Running into a problem viewing this blog post here (UnexpectedToken). Are you able to see this post? Removing it seems to work.

File: _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx [hunk 2]
```diff
@@ -7,102 +7,65 @@ authors: ["Prisha Adya"]
...
+			During my first year in forensics, we trained with a role-play scenario in which we investigated a crime scene and identified specific details such as time of death, how the person was murdered (based on a blood stain sample), and if the person had any type of medication or drugs in their system at the time of murder (through a post - mortem report).
+			During my time in the HOSA bowl category, the amount of information given was amazing.
+			From careers to history in healthcare to specific diseases of every part of the body and how to treat them, the learning opportunity was HUGE!
+			The amount of experience you gain is invaluable.</p>
+	<div align="center">
+			<img src="/blog/january-2021/chalk-outline.png" alt="Chalk outline of a body" width="500"/>
+	</div>
+	<div align="center">
+			<img src="/blog/january-2021/why-you-should-join-hosa/medical-demo.png" alt="HOSA event medical demo" width="500"/>
+	</div>
```
Comment (hirundinidae) [inline _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx:56]: Did we want to use `<figure>` around all the images? Some of the images on other blog posts use `<figcaption>`. Up to you (keep as `` or add `<figure>` for styling later)

**Review by hirundinidae — COMMENTED** [review]
Looks mostly good, there's one blog post I wasn't able to see without removing the comment. Otherwise, the HTML renders for me :]

File: _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx (hunk 2 above)
Comment (realsimonwu) [reply _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx:56]: The images from this particular blogpost don't have any captions or other things that would necessitate `<figure>` - I'll probably keep suing the `` tag here.

File: _digest/january-2021/will-ai-drive-healthcare-workers-into-extinction.mdx (hunk 1 above)
Comment (realsimonwu) [reply _digest/january-2021/will-ai-drive-healthcare-workers-into-extinction.mdx]: It's also preventing rendering for me - didn't catch it before. I've removed the comment in commit 38fc47d9

File: _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx (hunk 2 above)
Comment (hirundinidae) [reply _digest/january-2021/5-reasons-you-should-join-hosa-if-you-plan-to-go-into-medicine.mdx:56]: Yep, sounds good. The `<figure>` images are also wrapped with div that centers, don't think there will be any more styling on them.

**Review by hirundinidae — APPROVED** [review]
:+1:

**Pull request merged**